packages/adapter-pg/src/pg.ts

```diff
@@ -183,8 +183,12 @@ export class PrismaPgAdapter extends PgQueryable<StdClient> implements SqlDriver
   }
 
   async executeScript(script: string): Promise<void> {
-    // TODO: crude implementation for now, might need to refine it
-    for (const stmt of script.split(';')) {
+    const statements = script
+      .split(/;[\t\r\n ]*/)
```

Comment:
one more simplification:

```suggestion
      .split(/;\s*/)
```

(see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Character_class_escape#s)